CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+- `cortx-test`: An internal development-focused branch to stage breaking changes, while allowing the larger community to test CORTX in motion.
+- `CORTX-12345...`: Branches with the format of `CORTX-12345` are feature branches that are actively being worked upon by contributors and will eventually be PR'ed into `integration` once complete. Due to historical changes in tooling, feature branches may also begin with `EOS` or `UDX` as well. 
+
+## Contribution Process
+
+### Prerequisites
+
+The following prerequisites are all available in the root [CORTX](https://github.com/Seagate/cortx) repository:
+
+- Please read the [CORTX Code Style Guide](https://github.com/Seagate/cortx/blob/main/doc/CodeStyle.md).
+- Get started with [GitHub Tools and Procedures](https://github.com/Seagate/cortx/blob/main/doc/GitHub_Processes_and_Tools.rst), if you are new to GitHub.
+   - Please find additional information about [working with git](https://github.com/Seagate/cortx/blob/main/doc/working_with_git.md) specific to CORTX.
+- Please read about our [DCO and CLA policies](https://github.com/Seagate/cortx/blob/main/doc/dco_cla.md) in the root [CORTX](https://github.com/Seagate/cortx) repository.
+
+### Submitting a PR
+
+This compact flow for submitting a Pull Request to the [cortx-k8s](https://github.com/Seagate/cortx-k8s) repository will help ensure that it is able to be accepted and merged with minimal overhead or editing. 
+
+> :warning: The target branch of your Pull Request should be **`integration`** (as documented below).
+
+1. Fork [this repository](https://github.com/Seagate/cortx-k8s) to another personal or organization account on GitHub.
+2. Create a new development branch off of the upstream (also known as the original repository's) `integration` branch.
+   - `git checkout -b <my-new-feature-branch> integration`
+3. Do your work, including writing code, writing tests, updating documentation, and passing all tests locally.
+4. Fetch the latest upstream changes to ensure you have the latest working copy of the upstream codebase.
+   - `git fetch upstream`
+5. Rebase your changes on top of the latest upstream streams, being sure to resolve any conflicts. _(This process allows you to resolve any local issues with your changes, one by one, before submitting your PR instead of afterwards.)_
+   - `git branch --set-upstream-to=upstream/integration`
+   - `git rebase`
+6. Push your rebased change set to your repository.
+   - `git push origin <my-new-feature-branch>`
+7. Create a pull request from your `<my-new-feature-branch>` against the upstream **`integration`** branch.
+   1. _**NOTE:**_ The target branch of your Pull Request should be `integration`, not `main`!
```

Comment:
I suggest we switch the default branch to `integration`, which makes the default base branch for PRs `integration`. This will make it easy for those submitting PRs, and avoid accidentally targeting main. It also means we don't have to stress this point so much, it can just be the single statement in item 7.

The default branch does change which branch is shown by default on GitHub, and the initial branch that is checked out when you clone. I could argue using integration is better for those cases as well.

Reply:
I agree. We'll want to communicate this change so that others are aware, although I believe it should be a minimal impact overall if other automation is running against version-specific builds.